src/cryptoadvance/specter/templates/wallet/wallets_overview.jinja

```diff
@@ -25,7 +25,7 @@
         <small style="line-height:30px">Total balance:</small><br>
         <span style="color: #fff">
 			{% set fullbalance = specter.wallet_manager.wallets.values() | sum(attribute='fullbalance') %}
-            {{ fullbalance | btcunitamount }}
+            <div id="fullbalance_amount">{{ fullbalance | btcunitamount }}</div>
```

Comment:
```suggestion
            <span id="fullbalance_amount">{{ fullbalance | btcunitamount }}</span>
```
div causes the BTC to go to new line, should be either a span or include the BTC/ sats in the div.